### PR TITLE
feat(api): Also expose the core ORT version as part of the endpoint

### PR DESCRIPTION
**Pull request by sschuberth**

### CODE REVIEW

File: core/src/main/kotlin/api/VersionsRoute.kt [hunk 1]
```diff
@@ -28,10 +28,13 @@ import io.ktor.server.routing.Route
...
 import org.eclipse.apoapsis.ortserver.model.ORT_SERVER_VERSION
 
+import org.ossreviewtoolkit.utils.ort.Environment
+
 fun Route.versions() = get("versions", getVersions) {
     call.respond(
         mapOf(
-            "ORT Server" to ORT_SERVER_VERSION
+            "ORT Server" to ORT_SERVER_VERSION,
+            "ORT Core" to Environment.ORT_VERSION
```
Comment (mnonnenmacher) [inline core/src/main/kotlin/api/VersionsRoute.kt:37]: Should this be called "ORT Core"? I think we use "Core" nowhere else in the context of ORT. How about just "ORT" or spell it out to "OSS Review Toolkit"?

**Review by mnonnenmacher — CHANGES_REQUESTED** [review]

File: core/src/main/kotlin/api/VersionsRoute.kt (hunk 1 above)
Comment (sschuberth) [reply core/src/main/kotlin/api/VersionsRoute.kt:37]: > I think we use "Core" nowhere else in the context of ORT.

Not until now at least, but the need arose with the advent of ORT Server. I somehow wanted to mention "ORT", but at the same time make clear that this is not ORT, the CLI, that people are used to refer to, but the sum of the ORT libraries.

So, I think it's justified to "invent" the new term "ORT Core" here. @mmurto any opinion?

Comment (sschuberth) [reply core/src/main/kotlin/api/VersionsRoute.kt:37]: Among these options, I'd prefer to stick with "ORT Core", and "ORT" goes second.

**Review by mnonnenmacher — APPROVED** [review]

File: core/src/main/kotlin/api/VersionsRoute.kt (hunk 1 above)
Comment (mmurto) [reply core/src/main/kotlin/api/VersionsRoute.kt:37]: Just popped in to mind that "ORT Core" may be a little confusing sometimes as the API component of ORT Server is also called "Core".

**Pull request merged**